.github/SwiftCurrentLint/.swiftlint.yml

```diff
@@ -133,8 +133,8 @@ included: # paths to include during linting. `--path` is ignored if present.
 #   - Workflow/Package.swift
 
 line_length:
```

Comment:
As discussed, I think I'm fine dropping this.

Reply:
If we do drop this, we should go through the code and remove the unnecessary suppressions (though we'll get warnings for that anyways).

Reply:
We'll do this in another PR because this one is already massive and we don't want to hide this change.